src/multibody/joint/joint-revolute-unaligned.hpp

```diff
@@ -342,12 +339,8 @@ namespace se3
 	       const Eigen::VectorXd & qs ) const
     {
       const double & q = qs[idx_q()];
-
-      /* It should not be necessary to copy axis in jdata, however a current bug
-       * in the fusion visitor prevents a proper access to jmodel::axis. A
-       * by-pass is to access to a copy of it in jdata. */
-      data.angleaxis.angle() = q;
-      data.M.rotation(data.angleaxis.toRotationMatrix());
+      
+      data.M.rotation(Eigen::AngleAxisd(q, axis).toRotationMatrix());
```

Comment:
I see no reasons not to keep an initialized AngleAxis in JData. 
Possibly add an assert to double check that angleaxis.axis == jmodel.axis (I guess not an isApprox but an exact ==).